src/compas_timber/connections/l_halflap.py

```diff
@@ -1,14 +1,16 @@
-from compas.geometry import Frame
+from compas.tolerance import TOL
 
-from compas_timber.elements import CutFeature
-from compas_timber.elements import MillVolume
+from compas_timber._fabrication import JackRafterCut
+from compas_timber._fabrication import Lap
+from compas_timber.connections.utilities import beam_ref_side_incidence
+from compas_timber.connections.utilities import beam_ref_side_incidence_with_vector
 
 from .joint import BeamJoinningError
-from .lap_joint import LapJoint
+from .joint import Joint
 from .solver import JointTopology
 
 
-class LHalfLapJoint(LapJoint):
+class LHalfLapJoint(Joint):
```

Comment:
so are we sure we want to get rid of the `LapJoint` class? discussed with @obucklin as well?